test2code/src/main/kotlin/com/epam/drill/test2code/coverage/CoverageSender.kt

```diff
@@ -115,3 +107,22 @@ class IntervalCoverageSender(
         }
     }
 }
+
+private fun processTotalSizeByte(logger: KLogger): BigInteger {
```

Comment:
`processTotalSizeByte` is not the best name here:
- `process` it implies `processing` - like a thing that has an input, inner transformations and output.
- `...TotalSizeByte` - w/o context of a queue class its ambiguous

I suggest renaming to `getMaxQueueByteSize`

Reply:
changed